### DISCUSSION TITLE
Help me

### DISCUSSION THREAD

**Discussion by zxcqwe77** [SebaUbuntu/TWRP-device-tree-generator#42] [Q&A]
I am a newbie and have no idea where to write the path to recovery, can you please help me

**Comment by SebaUbuntu** [comment]
You should be able to drag and drop your recovery image into the terminal window, try that

**Comment by tkn7497** [comment]
![IMG_20210407_023911](https://user-images.githubusercontent.com/61553345/113790751-c0072580-974a-11eb-8b6b-7850934fa399.jpg)
I am getting an error like in the image. What is the reason? I want to make the operating system of my device android 11 twrp please can you help me.

**Reply by SebaUbuntu** [reply]
Latest commit broke the script, for now use PyPi version (follow the README)
